pkg/monitor/datapath_drop.go

```diff
@@ -34,38 +37,53 @@ type DropNotify struct {
 	Hash     uint32
 	OrigLen  uint32
 	CapLen   uint32
-	SrcLabel uint32
-	DstLabel uint32
+	SrcLabel identity.NumericIdentity
+	DstLabel identity.NumericIdentity
 	DstID    uint32
 	Unused   uint32
 	// data
 }
 
+// dumpIdentity dumps the source and destination identities in numeric or
+// human-readable format.
+func (n *DropNotify) dumpIdentity(buf *bufio.Writer, numeric bool) {
+	if numeric {
+		fmt.Fprintf(buf, "identity %d->%d", n.SrcLabel, n.DstLabel)
```

Comment:
(This comment applies elsewhere too)

`buf` can be the more generic `io.Writer` here, and elsewhere. Using the `io.Writer` interface as a destination rather than a specific type is used consistently throughout the codebase.